CHANGELOG-4.0.md

```diff
@@ -1,6 +1,8 @@
 CHANGELOG for 4.0.x
 ===================
 
+* feature #25091 [Security] Add target user to SwitchUserListener (jwmickey)
+
```

Comment:
No need to add this, will be done automatically when the version is released after merging.

Reply:
@iltar is right. But you need to add the feature to the component's changelog instead.

Reply:
I will move the changelog note to the correct file, thank you.